bezier/src/main/java/io/channel/bezier/compose/component/avatar/BezierAvatar.kt

```diff
@@ -0,0 +1,180 @@
+package io.channel.bezier.compose.component.avatar
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.requiredSize
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberAsyncImagePainter
+import io.channel.bezier.BezierTheme
+import io.channel.bezier.compose.R
+import io.channel.bezier.compose.component.avatar.properties.BezierAvatarBadge
+import io.channel.bezier.compose.component.avatar.properties.BezierAvatarSize
+import io.channel.bezier.compose.component.badge.chat.BezierChatBadge
+import io.channel.bezier.compose.component.badge.status.BezierStatusBadge
+import io.channel.bezier.extension.outlineBorder
+import io.channel.bezier.extension.thenIf
+import io.channel.bezier.shape.SmoothRoundedCornerShape
+
+private const val AvatarRadiusFraction = 42
+
+@Composable
+fun BezierAvatar(
+        url: String,
+        avatarSize: BezierAvatarSize,
```

Comment:
이전 뱃지와 동일하게 size로 하면 어떠신가요?
```suggestion
        size: BezierAvatarSize,
```